app/routes.py

```diff
@@ -1,2 +1,44 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify
 
+class Planet():
+    def __init__(self, id, name, description, rings):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.rings = rings
+
+    def to_json(self):
```

Comment:
👍 Nice work adding this helper. I'd suggest naming it `to_dict` rather than `to_json`, since that's really what it's doing. Flask still needs to `jsonify` the result to send it over the network, and it can be confusing to have code like `return jsonify(planet.to_json())` since it raises the question of why do we jsonify if the data is already json?